src/main/java/org/sagebionetworks/bridge/services/StudyActivityEventService.java

```diff
@@ -228,6 +234,7 @@ public void publishEvent(StudyActivityEvent event, boolean showError, boolean up
         for (String fieldName : GLOBAL_EVENTS_OF_INTEREST) {
             addIfPresent(events, map, fieldName, true);    
         }
+        events.sort(Comparator.comparing(StudyActivityEvent::getEventId));
         return new ResourceList<>(events); 
     }
     
```

Comment:
We need tests for the SQL query, also verify that a study burst event when updated copies over the study burst metadata to the event that is persisted.